data/json/items/melee.json

```diff
@@ -2333,5 +2333,46 @@
     "techniques": "RAPID",
     "qualities": [ [ "CUT", 1 ], [ "CUT_FINE", 1 ], [ "BUTCHER", 7 ] ],
     "flags": [ "STAB", "SHEATH_KNIFE" ]
-  }
+  },
+  {
+    "id": "butterfly_swords",
+    "type": "TOOL",
+    "symbol": "/",
+    "color": "light_gray",
+    "name": "butterfly swords",
+    "name_plural": "butterfly swords",
+    "description": "This is a matched pair of traditional Shaolin butterfly swords.  They are about the size of machetes but have hand guards and wider blades.",
```

Comment:
I think implementing dual-wielding weapons like that is a huge hack. Would have to be one weapon.

Reply:
Right, this was knowingly done as a hack that could latter to made into real dual-wielding. This will do for now?

Reply:
I suggest change the name into "pair(s) of butterfly swords", to avoid confusing phrases such as "2 butterfly swords" which may actually be 2 pairs.

Reply:
Sounds good.